lib/app/item.ex

```diff
@@ -48,7 +48,6 @@ defmodule App.Item do
   def create_item_with_tags(attrs) do
     %Item{}
     |> changeset_with_tags(attrs)
-    |> IO.inspect()
```

Comment:
worth updating the `Elixir` version to `v1.14` so that we can use [`dbg`](https://github.com/elixir-lang/elixir/blob/v1.14/CHANGELOG.md#dbg)? 😉 `#finally` 🎉